Mail/Views/Settings/SettingsView.swift

```diff
@@ -47,15 +49,62 @@ struct SettingsView: View {
     @AppStorage(UserDefaults.shared.key(.matomoAuthorized)) private var matomoAuthorized: Bool = DefaultPreferences
         .matomoAuthorized
 
+    @State private var isShowingMyKSuiteDashboard = false
+    @State private var myKSuiteMailbox: Mailbox?
+
     var body: some View {
         ScrollView {
             VStack(alignment: .leading, spacing: 0) {
+                // MARK: - Section: my kSuite
+
+                if mainViewState.mailboxManager.mailbox.isFree {
+                    Group {
+                        SettingsSectionTitleView(title: "my kSuite")
+
+                        if let myKSuiteMailbox, let mailboxManager = accountManager.getMailboxManager(for: myKSuiteMailbox) {
+                            SettingsSubMenuCell(title: myKSuiteMailbox.email) {
+                                MailboxSettingsView(mailboxManager: mailboxManager)
+                            }
+                        }
+
+                        SettingsSubMenuLabel(title: MailResourcesStrings.Localizable.myKSuiteSubscriptionTitle)
+                            .onTapGesture {
+                                isShowingMyKSuiteDashboard = true
+                            }
+                            .sheet(isPresented: $isShowingMyKSuiteDashboard) {
+                                MyKSuiteDashboardView(
+                                    apiFetcher: mainViewState.mailboxManager.apiFetcher,
+                                    userId: currentUser.value.id
+                                ) {
+                                    AvatarView(
+                                        mailboxManager: mainViewState.mailboxManager,
+                                        contactConfiguration: .user(user: currentUser.value),
+                                        size: 24
+                                    )
+                                }
+                            }
+
+                        IKDivider()
+                    }
+                    .task {
+                        let configuration = {
+                            @InjectService var mailboxInfosManager: MailboxInfosManager
+                            return mailboxInfosManager.realmConfiguration
+                        }()
+                        let mailboxes = ObservedResults(Mailbox.self, configuration: configuration) {
+                            $0.userId == currentUser.value.id && $0.isFree
+                        }
+                        myKSuiteMailbox = mailboxes.wrappedValue.first
+                    }
+                }
```

Comment:
We have to hide the entry point if we don't have a kSuite. We will discuss it tomorrow